### PR TITLE
update `reduction.h`

### PR DESCRIPTION
**Pull request by Li-Wen-Di**
取消了m12这种形式，对于general和Colinear情况的两圈IBP都写成了procedure的形式，对于单圈IBP两种情况公用一个procedure

### CODE REVIEW

File: ext/reduction.h [hunk 1]
```diff
@@ -83,19 +50,23 @@
 #endprocedure
 
 
-
-#procedure CollinearTwoLoopRecursion
+*** 对Colinear情况做IBP
+#procedure ColinearTwoLoopRecursion
```
Comment (Fenyutanchan) [inline ext/reduction.h]: Should be "Collinear" instead of "Colinear".

File: src/FORM_scripts.jl [hunk 2]
```diff
@@ -50,15 +50,16 @@ function __collinear_form_script(
...
+    Symbol a1, a2, a3, m1, m2, m3, D, invdelta;
 
     set Pos2ToN: 2,...,$(num_do_loop);
 
     Off Statistics;
 
     Local I = $(integral);
 
-    #call CollinearTwoLoopRecursion
+    #call ColinearTwoLoopRecursion
```
Comment (Fenyutanchan) [inline src/FORM_scripts.jl]: Should be "Collinear" instead of "Colinear".

**Review by Fenyutanchan — CHANGES_REQUESTED** [review]
Typo should be improve.

**Review by Fenyutanchan — APPROVED** [review]
Done.

**Comment by Li-Wen-Di** [comment]
Thanks

**Pull request merged**